admin_guide/out_of_resource_handling.adoc

```diff
@@ -42,8 +42,7 @@ xref:../admin_guide/overcommit.adoc#disabling-swap-memory[disable swap].
 An _eviction policy_ allows a node to fail one or more pods when the node is running low on available resources.
 Failing a pod allows the node to reclaim needed resources.
 
-An eviciton policy is a combination of an xref:out-of-resource-eviction-signals[eviction trigger signal] with a specific xref:out-of-resource-eviction-thresholds[eviction threshold value], that is set
-in the node configuration file or through the xref:out-of-resource-eviction-thresholds[command line]. Evictions can be either hard, where a node takes immediate action on a pod that exceeds a threshold, or soft, where a node allows a grace period before taking action. See the sections below for important information about the differences between xref:out-of-resource-eviction-thresholds[hard and soft evictions].
+An eviction policy is a combination of an xref:out-of-resource-eviction-signals[eviction trigger signal] with a specific xref:out-of-resource-eviction-thresholds[eviction threshold value], that is set in the node configuration file or through the xref:out-of-resource-eviction-thresholds[command line]. Evictions can be either hard, where a node takes immediate action on a pod that exceeds a threshold, or soft, where a node allows a grace period before taking action. See the sections below for important information about the differences between xref:out-of-resource-eviction-thresholds[hard and soft evictions].
```

Comment:
no comma after [eviction threshold value]

Reply:
I'd remove "See the sections below for important information about the differences between xref:out-of-resource-eviction-thresholds[hard and soft evictions]." and put the xref on "Evictions" at the start of the previous sentence.